orchestrator/services/watch_actions.py

```
@@ -0,0 +1,746 @@
+            db, run=run, new_state=RunState.RUNNING,
+            actor_type=ActorType.COORDINATOR, actor_id="watcher",
+            reason="Watch reassign complete -- resuming",
+        )
+        transition_task(
+            db, task=clone, new_state=TaskState.QUEUED,
+            actor_type=ActorType.COORDINATOR, actor_id="watcher",
+            reason=f"Queued for '{chosen_name}' after watch reassign",
+        )
+        try:
+            create_task_board_task(db, run, clone)
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)